.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] I think it could be a good idea to also ignore the node_modules folder for better practices since by mistake you could upload it and it is not actually necessary. Please, ignore the node modules folder, you could find how [here](https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/) :memo: :smiley: